tests/probes/yamlfilecontent/test_probes_yamlfilecontent.xml

```diff
@@ -0,0 +1,136 @@
+
+    <definition class="compliance" version="1" id="oval:0:def:2"> <!-- comment="error" -->
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="non_scalar_get" test_ref="oval:0:tst:5"/>
+      </criteria>
+    </definition>
+
+  </definitions>
+
+  <tests>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:1" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:1"/>
+      <ind-def:state state_ref="oval:0:ste:1"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:2" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:2"/>
+      <ind-def:state state_ref="oval:0:ste:2"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:3" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:3"/>
+      <ind-def:state state_ref="oval:0:ste:3"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:4" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:4"/>
+      <ind-def:state state_ref="oval:0:ste:4"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:5" check="all" comment="error">
+      <ind-def:object object_ref="oval:0:obj:5"/>
+    </ind-def:yamlfilecontent_test>
+
+  </tests>
+
+  <objects>
+
+    <ind-def:yamlfilecontent_object version="1" id="oval:0:obj:1">
+      <ind-def:path>/tmp</ind-def:path>
+      <ind-def:filename>openshift-logging.yaml</ind-def:filename>
+      <ind-def:yamlpath>/kind</ind-def:yamlpath>
+      <ind-def:instance operation="greater than or equal" datatype="int">0</ind-def:instance>
+    </ind-def:yamlfilecontent_object>
+
+    <ind-def:yamlfilecontent_object version="1" id="oval:0:obj:2">
+      <ind-def:path>/tmp</ind-def:path>
+      <ind-def:filename>openshift-logging.yaml</ind-def:filename>
+      <ind-def:yamlpath>/metadata/namespace</ind-def:yamlpath>
+      <ind-def:instance operation="greater than or equal" datatype="int">0</ind-def:instance>
+    </ind-def:yamlfilecontent_object>
+
+    <ind-def:yamlfilecontent_object version="1" id="oval:0:obj:3">
+      <ind-def:path>/tmp</ind-def:path>
+      <ind-def:filename>openshift-logging.yaml</ind-def:filename>
+      <ind-def:yamlpath>/spec/outputs[type=forward]/name</ind-def:yamlpath>
+      <ind-def:instance operation="greater than or equal" datatype="int">0</ind-def:instance>
+    </ind-def:yamlfilecontent_object>
+
+    <ind-def:yamlfilecontent_object version="1" id="oval:0:obj:4">
+      <ind-def:path>/tmp</ind-def:path>
+      <ind-def:filename>openshift-logging.yaml</ind-def:filename>
+      <ind-def:yamlpath>/spec/outputs[type=elasticsearch]/name</ind-def:yamlpath>
+      <ind-def:instance operation="greater than or equal" datatype="int">0</ind-def:instance>
```

Comment:
I can't see `instance` element in the schema in https://github.com/OVAL-Community/OVAL/pull/90.

Reply:
I designed it after `xmlfilecontent` probe. There is no instance as well. I guess we don't need it because we can distinctively enumerate all the nodes in the YAML document, unlike `filecontent` probe.

Reply:
`instance` is also about selecting how many or which collected objects should be evaluated in the test.
If a yamlpath yields two values, like this test would, should it evaluate to error?

Reply:
How `xmlfilecontent` is able to return *a list of text strings* without it?

Reply:
I misinterpreted the `instance` field., it can return multiple values, it just won't let you choose the objects.

Reply:
I'll remove the `instance`.

Reply:
great!